src/main/java/seedu/guest/model/guest/DateRange.java

```diff
@@ -0,0 +1,65 @@
+package seedu.guest.model.guest;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.guest.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.format.ResolverStyle;
+
+/**
+ * Represents the dates that a Guest is staying at the hotel for.
+ * Guarantees: immutable; is valid as declared in {@link #isValidDateRange(String)}
+ */
+public class DateRange {
+    public static final String MESSAGE_CONSTRAINTS =
+            "DateRanges should contain only numbers, hyphens, backslashes, and spaces. They must follow the format"
+                    + "\"dd/MM/yy - dd/MM/yy\" and the second date must be later than the first date.";
+    private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("dd/MM/uu")
+            .withResolverStyle(ResolverStyle.STRICT);
+    public final String value;
+
+    /**
+     * Constructs a {@code CheckInDate}.
```

Comment:
Should this JavaDocs be constructing a date range?

Reply:
Yes, fixed